src/main/java/eu/execom/pomodoroTeam/PomodoroTeamApplication.java

```diff
@@ -2,11 +2,15 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.core.annotation.Order;
 
 @SpringBootApplication
+@Order(1000)  
 public class PomodoroTeamApplication {
-
-	public static void main(String[] args) {
+	
+	 public static void main(String[] args) {
 		SpringApplication.run(PomodoroTeamApplication.class, args);
 	}
+	
```

Comment:
Please make sure that the formatter is active and that 4 spaces are inserted instead of a tab. Using tabs can create the impression that the code is messy.